### PR TITLE
Plans: simple fix because analytics is business only

### PR DESCRIPTION
**Pull request by rralian**
/cc @artpi

### CODE REVIEW

File: client/my-sites/stats/stats-countries/index.jsx [hunk 1]
```diff
@@ -133,7 +133,7 @@ export default React.createClass( {
 								{ this.props.summary &&
 									<UpgradeNudge
 										title="Add Google Analytics"
-										message="Upgrade to Premium for Google Analytics integration."
+										message="Upgrade to Business for Google Analytics integration."
```
Comment (retrofox) [inline client/my-sites/stats/stats-countries/index.jsx]: should we use translation here?

Comment (rralian) [inline client/my-sites/stats/stats-countries/index.jsx]: yes, I'll fix both instances in this PR.

**Comment by rralian** [comment]
translations added

**Comment by retrofox** [comment]
:ship:

**Pull request merged**